saltstack/base/salt/luks/bin/populate-luks-volume.sh

```diff
@@ -157,12 +157,23 @@ process_freeipa_locations() {
   fi
 }
 
+process_cm_locations() {
+  if [[ "$IS_FREEIPA" != "true" ]]
+  then
+    log_processing_needed "Cloudera Manager"
+    /opt/cloudera/cm/bin/change_secrets_storage.sh "$MOUNT_POINT"
```

Comment:
I know CM folks added this [OPSAPS-69120](https://jira.cloudera.com/browse/OPSAPS-69120), but it does not currently bring any benefits over invoking `process_location` for every location involved. Looks like that script places everything into the subtree `$SECRET_PATH/cm` instead of `$SECRET_PATH`, which is inconsistent with the directory structure we use here for the other locations.

I thus wonder if we would be better off skip using the CM script and relying on `process_location` instead? (The parent paths `/etc` and `/var/lib` always exist, so no need to use `create_location_if_necessary` or `process_location_with_create`.)